TWLight/applications/views.py

```diff
@@ -1556,10 +1556,11 @@ def get_object(self):
             messages.add_message(
                 self.request,
                 messages.WARNING,
+                # fmt: off
```

Comment:
Please remove the `# fmt: off` and `# fmt: on` comments.

Reply:
```
messages.add_message(
                self.request,
                messages.WARNING,
                # Translators: This message is displayed when an attempt by a user to renew an application has been denied for some reason.
                _("Attempt to renew unapproved application #{pk} has been denied")
                .format(pk=app.pk),
)
```
If I keep it like this, then the lint test fails and tells to reformat the above block of code into this:-

```
messages.add_message(
                self.request,
                messages.WARNING,
                # Translators: This message is displayed when an attempt by a user to renew an application has been denied for some reason.
                _(
                    "Attempt to renew unapproved application #{pk} has been denied"
                ).format(pk=app.pk),
            )
```

And if I keep the reformatted block of code and then I run test then its show this error.
```
ERROR: ugettext messages should't be preceded by a newline
TWLight/applications/views.py:1561
                    "Attempt to renew unapproved application #{pk} has been denied"

```

I checked some different messages in that file and some of them were using those comments like this one.
[TWLight/applications/views.py#L1530](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/applications/views.py#L1530)

Reply:
Got it! I think then you should keep the black formatting comments, then. Thanks for letting me know